ranker/shared/ranker_obj.py

```diff
@@ -15,12 +16,11 @@
 class Ranker:
     """Ranker."""
 
-    DEFAULT_WEIGHT = 1e-2
 
     def __init__(self, message, profile = "blended"):
         """Create ranker."""
-        self.kgraph = message.get("knowledge_graph", {"nodes": {}, "edges": {}})
-        self.qgraph = message.get("query_graph", {"nodes":{}, "edges":{}} )
+        self.kgraph = message['knowledge_graph']
+        self.qgraph = message['query_graph']
```

Comment:
Is this going to fail when e.g. the incoming message
1. doesn't have a knowledge graph
2. has "knowledge_graph": None
3. has "knowledge_graph": {}
Or is pydantic taking care of all of these cases?

Reply:
Seems like this was outdated code that somehow got back in. It's now fixed.